tst/EnergyPlus/unit/SolarShading.unit.cc

```diff
@@ -3879,9 +3879,21 @@ TEST_F(EnergyPlusFixture, SolarShadingTest_Warn_Pixel_Count_and_TM_Schedule)
 
     SolarShading::GetShadowingInput(*state);
 
-    EXPECT_EQ(state->dataErrTracking->TotalWarningErrors, 0);
-    // Now expect one severe error from GetShadowInput()
-    EXPECT_EQ(state->dataErrTracking->TotalSevereErrors, 1);
-    // There should be a severe warning reported about the PixelCounting and the scheduled shading surface tm values > 0.0 combination.
-    EXPECT_EQ(state->dataErrTracking->LastSevereError, "The Shading Calculation Method of choice is \"PixelCounting\"; ");
+#ifdef EP_NO_OPENGL
```

Comment:
OK, so if OpenGL is not available, EnergyPlus issues a single warning to indicate polygon clipping will be used, and the simulation runs without pixel counting.  This explains why you have a single warning and no severe errors here.